### PR TITLE
Bug fix when Draw is created and Control added to Map

### PR DESCRIPTION
**Pull request by ericj974**
Bug fix for Control when adding to map

### CODE REVIEW

File: pyqtlet2/leaflet/layer/marker/marker.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def _onMove(self, event):
         self._logger.debug('marker moved. event: {event}'.format(event=event))
         event = self._qJsonValueToDict(event)
         self.latLng = [event["latlng"]["lat"], event["latlng"]["lng"]]
-        self.move.emit(event)
+        self.move.emit({**self._qJsonValueToDict(event), "latLng": self.latLng, "sender": self})
```
Comment (JaWeilBaum) [inline pyqtlet2/leaflet/layer/marker/marker.py:31]: The `latLng` values are already contained in the event, to reduce redundent values, I would suggest to leave the `latLng` field out. 
```suggestion
        self.move.emit({**self._qJsonValueToDict(event), "sender": self})
```

**Review by JaWeilBaum — COMMENTED** [review]
Thanks for the contribution. I never looked into the `Control` part of the library. Thanks for already adjusting to the mapIndex part.

**Comment by ericj974** [comment]
Ok, accepted your updates to leave Marker._onMove unchanged.

**Comment by JaWeilBaum** [comment]
This is interesting - did not come a cross this yet... My fault here!
Definetly this should be consitent as well!

**Review by JaWeilBaum — APPROVED** [review]

**Comment by JaWeilBaum** [comment]
@ericj974 I'm sorry for this. Just checkd some projects which use these events, all of them rely on `latLng` and I'll adjust to what you proposed!

**Comment by ericj974** [comment]
Ok @JaWeilBaum, otherwise I can do it. Thanks for being responsive, actively using your repo :)

**Comment by JaWeilBaum** [comment]
Always happy to see people using it and working on it as well. Version `0.9.1` should be available now :)

**Pull request merged**